origin.spec

```diff
@@ -102,9 +103,14 @@ Summary:        %{product_name} Kubernetes server commands
 Requires:       util-linux
 Requires:       socat
 Requires:       iptables
-Provides:       hyperkube
-Provides:       atomic-openshift-hyperkube
-Provides:       atomic-openshift-node
+Provides:       hyperkube = %{version}
+Provides:       atomic-openshift-hyperkube = %{version}
+Provides:       atomic-openshift-node = %{version}
+%if 0%{?epel}
+Conflicts:      okd-hyperkube
+%else
+Conflicts:      openshift-hyperkube
```

Comment:
Why add this?

Reply:
Intention was to signal that the user can't install both at the same time due to conflicting files.

However, on second thought, as the openshift-hypershift rpm is not distributed stand-alone, I'd be fine with removing it again altogether because they might not ever ever come across each other.
Should I remove?

Reply:
@LorbusChris @smarterclayton beat me on this ;) after our chat over the weekend i started to dig a bit and found https://github.com/openshift/release/pull/4466 & https://github.com/openshift/installer/pull/2131 hence the hint i guess.

Reply:
Above links have something to do with removing `hypershift`.  This is about `hyperkube`, though. I don't have a strong opinion about this one and can remove it, but my interpretation of https://docs.fedoraproject.org/en-US/packaging-guidelines/Conflicts/#_implicit_conflicts would still be to add it.

Reply:
sorry, ignore what i've said, monday morning w/o coffee.

Reply:
> This is about `hyperkube`, though. I don't have a strong opinion about this one and can remove it, but my interpretation of https://docs.fedoraproject.org/en-US/packaging-guidelines/Conflicts/#_implicit_conflicts would still be to add it.

+ 1

Is there any disadvantage to being verbose and including the conflict in this situation?

Reply:
In this case we shouldn't have `atomic-openshift` anymore, OKD can use `openshift` for package names.

Reply:
`openshift-hyperkube`